CMakeLists.txt

```diff
@@ -4,16 +4,13 @@ option(RGM_ENABLE_GRPC_SERVER "Enable the GRPC client plugin for compilation and
 option(RGM_BUILD_EMAKE "Build Emake and the compiler." ON)
 option(RGM_BUILD_STATIC "Build static libs." OFF)
 
-if (RGM_BUILD_STATIC)
+# FIXME: MSVC dynamic linking requires US TO DLLEXPORT our funcs
+# since we currently don't, I'm force disabling the option on MSVC
+if (RGM_BUILD_STATIC OR MSVC)
```

Comment:
it failed n tried to build dll for msvc

Reply:
Where are you actually using `LIB_TYPE` at? I don't see `LIB_TYPE` being used anywhere in here.

Reply:
it's used in all the included cmakefiles in the enigma pr

Reply:
Well there's no reason this shouldn't work, ask Josh. I don't believe this doesn't work.

Reply:
You can see it not working. in ci. Just put a dependent option as it's more correct anyways

Reply:
Alright dc8723e0c243047026623e0e0fe824f3ed52858c is sent, but I don't see how it will work.